src/utils/createFeatureStoreContent.js

```diff
@@ -75,7 +75,10 @@ const createFeatureSetsRowData = (featureSet, project) => {
       type: 'hidden'
     },
     entity: {
-      value: featureSet.entities ? featureSet.entities[0]?.name : '',
+      value: featureSet.entities[0]?.name
+        ? `${featureSet.entities[0]?.name} ${featureSet.entities[1]?.name ??
+            ''}`
+        : '',
```

Comment:
Please add comma as a separator.
Also, I would suggest:

```suggestion
      value:
        featureSet.entities
          ?.slice(0, 2)
          .map(entity => entity.name)
          .join(', ') ?? '',
```

For `featureSet.entities` of `undefined` it will return `''`.
For `featureSet.entities` of `[]` it will return `''`.
For `featureSet.entities` of `[{ name: 'a' }]` it will return `'a'`.
For `featureSet.entities` of `[{ name: 'a' }, { name: 'b' }]` it will return `'a, b'`.
For `featureSet.entities` of `[{ name: 'a' }, { name: 'b' }, { name: 'c' }]` it will return `'a, b'`.